generators/client/templates/angular/_package.json

```diff
@@ -138,6 +138,7 @@
     "node": ">=6.9.0"
   },
   "scripts": {
+    "code-format":"node ../generator-jhipster/cli/format.js; yarn lint:fix",
```

Comment:
change command name to `prettier:format` or `prettify` so that its easy for people to relate to prettier

Reply:
also dont use yarn directly use `<%= clientPackageManager %>`